src/devcenter/azext_devcenter/aaz/latest/devcenter/admin/pool/_create.py

```diff
@@ -87,26 +86,44 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.dev_box_definition_name = AAZStrArg(
             options=["-d", "--dev-box-definition-name"],
             arg_group="Properties",
-            help="Name of a Dev Box definition in parent Project of this Pool",
+            help="Name of a dev box definition in parent project of this pool.",
         )
         _args_schema.license_type = AAZStrArg(
             options=["--license-type"],
             arg_group="Properties",
-            help="Specifies the license type indicating the caller has already acquired licenses for the Dev Boxes that will be created.",
+            help="Specifies the license type indicating the caller has already acquired licenses for the dev boxes that will be created.",
             default="Windows_Client",
             enum={"Windows_Client": "Windows_Client"},
         )
         _args_schema.local_administrator = AAZStrArg(
             options=["--local-administrator"],
             arg_group="Properties",
-            help="Indicates whether owners of Dev Boxes in this pool are added as local administrators on the Dev Box.",
+            help="Indicates whether owners of dev boxes in this pool are added as local administrators on the dev box.",
             enum={"Disabled": "Disabled", "Enabled": "Enabled"},
         )
         _args_schema.network_connection_name = AAZStrArg(
             options=["-c", "--network-connection-name"],
             arg_group="Properties",
-            help="Name of a Network Connection in parent Project of this Pool",
+            help="Name of a network connection in parent project of this pool.",
         )
+
+        #TODO: Uncomment once idle is ready, uncomment other blocks of code below
+        # _args_schema.stop_on_disconnect = AAZObjectArg(
+        #     options=["--stop-on-disconnect"],
+        #     arg_group="Properties",
+        #     help="Stop on disconnect configuration settings for dev boxes created in this pool.",
+        # )
+
+        # stop_on_disconnect = cls._args_schema.stop_on_disconnect
+        # stop_on_disconnect.grace_period_minutes = AAZIntArg(
+        #     options=["grace-period-minutes"],
+        #     help="The specified time in minutes to wait before stopping a dev box once disconnect is detected.",
+        # )
+        # stop_on_disconnect.status = AAZStrArg(
+        #     options=["status"],
+        #     help="Whether the feature to stop the dev box on disconnect once the grace period has lapsed is enabled.",
+        #     enum={"Disabled": "Disabled", "Enabled": "Enabled"},
+        # )
```

Comment:
You can hidden this argument in aaz-dev-tools.

Reply:
Confirmed that the API name should be changed

Reply:
Fixed